contracts/epochs/FlowClusterQC.cdc

```diff
@@ -258,6 +258,21 @@ access(all) contract FlowClusterQC {
         }
     }
 
+    /// Represents the quorum certificate vote data for a signer 
+    /// of the certificate.
+    access(all) struct ClusterQCVoteData {
+        /// The aggregated signature, hex-encoded, encompasses all individual vote signatures contributed by nodes across the cluster
+        access(all) let aggregatedSignature: String
+
+        /// The node IDs that contributed their vote to the aggregated signature
+        access(all) let voterIDs: [String]
+
+        init(aggregatedSignature: String, voterIDs: [String]) {
+            self.aggregatedSignature = aggregatedSignature
```

Comment:
Are there any pre-conditions we can do here to verify anything?

Reply:
Probably just the length of the signature.

Reply:
Since the input needs to be machine-generated anyway, I think it has pretty low value.